pkgs/pinned.nix

```diff
@@ -6,13 +7,18 @@ let
   nixBitcoinPkgsUnstable = import ./. { pkgs = nixpkgsUnstable; };
 in
 {
-  inherit (nixpkgsUnstable)
+  # Pkgs from nixpkgs stable.
+  # Inherit from pkgs instead of nixpkgsStable to avoid evaluating another instance
+  # of nixpkgs
```

Comment:
So that means that every time we update nixpkgs we need to check if the version in unstable differs from stable? Not sure if that's worth the hassle since divergence may happen quickly for some packages. Somewhat independendently,
  1. For updating nixpkgs, I have local tooling now to diff the relevant nix expressions which helps writing release notes (and may help finding anomalies).
  2. I think it makes sense in the python tests to additionally check the versions of the packages (e.g. by running `bitcoind -version`) against some hardcoded strings in the python script. That way we can see from a nixpkgs update commit which of the relevant packages were updated.

Reply:
I've removed the commit which changes the pkgs pinning. I'll address this in a flakes PR which adds tooling to automate all aspects of nixpkgs updates.

Reply:
Ok, looking forward to your flakes PR. Sounds exciting!